firebase/test/firestore-rules/users/collections-executions.spec.ts

```diff
@@ -0,0 +1,33 @@
+import * as rulesTesting from "@firebase/rules-unit-testing";
+import * as utils from "../utils";
+
+describe("users/{userId}/collections_executions", () => {
+  const collectionId = (userId: string) => `users/${userId}/collections_executions`;
+  let collectionsExecutionsRef: utils.CollectionReference;
+  let firestore: utils.ClientFirestore;
+
+  describe("owner users", () => {
+    before(async () => {
+      firestore = await utils.createMyFirestore();
+      collectionsExecutionsRef = firestore.collection(collectionId(utils.myFirestoreUid));
+    });
+
+    it("should be able to read its collections executions", async () => {
+      await rulesTesting.assertSucceeds(collectionsExecutionsRef.get());
+    });
+
+    it("should be denied to read other collections executions", async () => {
+      const othersExecutions = firestore.collection(collectionId("any"));
+      await rulesTesting.assertFails(othersExecutions.get());
+    });
+
+    it("should be able to write its collections executions", async () => {
+      await rulesTesting.assertSucceeds(collectionsExecutionsRef.doc().set({ any: "any" }));
```

Comment:
this one you haven't added `.add` operations as we've added in other files

Reply:
Ditto: Add is the same here, because set adds if there is no existing document, which is the case.